### ISSUE TITLE
Doesnt recognise bfd variable

### ISSUE THREAD

**Issue by a-dhar** [terraform-google-modules/terraform-google-vpn#72]
### TL;DR

Example Code:

resource "google_compute_router_peer" "peer" {
  name                      = "my-router-peer"
  router                    = "my-router"
  region                    = "us-central1"
  peer_ip_address           = "169.xxx.x.x."
  peer_asn                  = 65513
  advertised_route_priority = 100
  interface                 = "interface-1"
  bfd                       = var.bfd
}
variables:

variable "bfd"{

 type = object({

   session_initialization_mode = string

   min_transmit_interval = number

   min_receive_interval  = number

   multiplier = number

 })

 default = {

   session_initialization_mode = "DISABLED"

   min_transmit_interval = null

   min_receive_interval  = null

   multiplier = null

 }

}

when we run a plan the error states:

 Error: Unsupported argument
│
│   on main.tf line 67, in resource "google_compute_router_peer" "peer":
│   67:   bfd                       = var.bfd
│
│ An argument named "bfd" is not expected here. Did you mean "id"?
╵

### Observed behavior

 Error: Unsupported argument
│
│   on main.tf line 67, in resource "google_compute_router_peer" "peer":
│   67:   bfd                       = var.bfd
│
│ An argument named "bfd" is not expected here. Did you mean "id"?
╵

### Terraform Configuration

```hcl
Example Code:

resource "google_compute_router_peer" "peer" {
  name                      = "my-router-peer"
  router                    = "my-router"
  region                    = "us-central1"
  peer_ip_address           = "169.xxx.x.x"
  peer_asn                  = 65513
  advertised_route_priority = 100
  interface                 = "interface-1"
  bfd                       = var.bfd
}
variables:

variable "bfd"{

 type = object({

   session_initialization_mode = string

   min_transmit_interval = number

   min_receive_interval  = number

   multiplier = number

 })

 default = {

   session_initialization_mode = "DISABLED"

   min_transmit_interval = null

   min_receive_interval  = null

   multiplier = null

 }

}

when we run a plan the error states:

 Error: Unsupported argument
│
│   on main.tf line 67, in resource "google_compute_router_peer" "peer":
│   67:   bfd                       = var.bfd
│
│ An argument named "bfd" is not expected here. Did you mean "id"?
╵
```

### Terraform Version

```sh
v1.0.7 and v1.1.2
```
